CHANGELOG.md

```diff
@@ -21,7 +21,115 @@ copy from UI changelog
 
 </details>
 
-next release v1.65.0 / v2.2.0 (2025-01-08)
+v1.66.0 / v2.3.0 (2025-02-03)
+-------------------------------
+### Backend Changes
+
+#### ⛔ Breaking Changes
+
+* [refactor] move cassandra implementation to `internal` package ([@mahadzaryab1](https://github.com/mahadzaryab1) in [#6639](https://github.com/jaegertracing/jaeger/pull/6639))
```

Comment:
@jkowall Sorry about this but would you be able to regenerate the changelog. I had the wrong tag selected for this PR. I fixed it now